content/influxdb/v2.0/get-started.md

```diff
@@ -28,9 +28,17 @@ _See [Differences between InfluxDB Cloud and InfluxDB OSS](#differences-between-
 
 <!-------------------------------- BEGIN macOS -------------------------------->
 {{% tab-content %}}
-### Download and install InfluxDB v2.0
+### Install InfluxDB v2.0
+
+We recommend installing InfluxDB v2.0 using [Homebrew](https://brew.sh/) on MacOS.
+With Homebrew installed, run:
+
+```sh
+brew update
+brew install influxdb
+```
```

Comment:
~~~suggestion
We recommend using [Homebrew](https://brew.sh/) to install InfluxDB v2.0 on macOS:

```sh
brew update
brew install influxdb
```
~~~